src/Post_Command.php

```diff
@@ -797,4 +819,22 @@ private function get_category_ids( $arg ) {
 		// If no category ids found, return exploded array for compat with previous WP-CLI versions.
 		return $category_ids ? $category_ids : $categories;
 	}
+
+      /**
+       * Get post metadata.
+       *
+       * @param $post_id id of the post.
+       * @return array
+       */
+      private function get_metadata( $post_id = null ) {
```

Comment:
Instead of making `$post_id` optional, we should make it required:

```
get_metadata( $post_id = null )
```

Reply:
sure.